docs/bot/configuration.md

```diff
@@ -21,7 +21,8 @@ These are the keys you can specify:
 | `files`                          | Array of files to update.                                                                           | Default: `['README.md']`                                                                                    |
 | `imageSize`                      | Size (in px) of the user's avatar.                                                                  | Default: `100`                                                                                              |
 | `commit`                         | Auto-commit badge when adding contributors.                                                         | Default: `false`                                                                                           |
-| `commitConvention`               | Commit convention ([`angular`](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines), [`atom`](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#git-commit-messages), [`ember`](https://guides.emberjs.com/v1.10.0/contributing/#toc_commits), [`eslint`](https://eslint.org/docs/1.0.0/developer-guide/contributing#step-2-make-your-changes), [`jshint`](https://jshint.com/contribute/), [`gitmoji`](https://gitmoji.carloscuesta.me/), or `none`).                                                         | Default: `angular`                                                                                           |
+| `commitConvention`               | Commit convention ([`angular`](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit), [`atom`](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#git-commit-messages), [`ember`](https://guides.emberjs.com/v1.10.0/contributing/#toc_commits), [`eslint`](https://eslint.org/docs/latest/contribute/pull-requests#step2), [`jshint`](https://jshint.com/contribute/), [`gitmoji`](https://gitmoji.carloscuesta.me/), or `none`).                                                         | Default: `angular`                                                                                           |
```

Comment:
@danieltott Hi. I think no need to change this line. Kindly remove it.

Reply:
@tenshiAMD there were two broken links in the original docs (angular and eslint) - hence the changes. I can revert them if you'd like, but then the links will still be broken

Reply:
> @tenshiAMD there were two broken links in the original docs (angular and eslint) - hence the changes. I can revert them if you'd like, but then the links will still be broken

I think it should be in a separate commit. However, it is not too important as those links came from third party and can change anytime. I think what's important are the options that the user will able to use.

Reply:
Fair enough - will create a separate PR for those 👍🏼